src/main/java/uk/gov/hmcts/reform/blobrouter/data/reconciliation/reports/ReconciliationReportRepository.java

```diff
@@ -81,12 +81,12 @@ public UUID save(NewReconciliationReport report) throws SQLException {
     public Optional<ReconciliationReport> getLatestReconciliationReport(LocalDate forDate, String account) {
         try {
             ReconciliationReport report = jdbcTemplate.queryForObject(
-                "SELECT * "
-                    + "FROM envelope_reconciliation_reports "
-                    + "WHERE account = :account"
-                    + "  AND DATE(created_at) = :date "
-                    + "ORDER BY created_at DESC "
-                    + "LIMIT 1",
+                "SELECT er.*"
```

Comment:
is this indentation change a manual one? ie, will it not jump back to what it was after automatically formatting this file?